atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepQueueCleaner.java

```diff
@@ -50,21 +50,29 @@ public void clean(ShardAndStrategy shardStrategy, long oldProgress, long newProg
     }
 
     private void cleanSweepableCells(ShardAndStrategy shardStrategy, long oldProgress, long newProgress) {
```

Comment:
This will now need to take in the SweepBatch, and extract the exact rows to clean up from there

Reply:
This is done on the part 3 branch